CairoSAC/SACEth003.xml

```diff
@@ -33,42 +32,467 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <dimensions type="margin" unit="mm">
+                                        <dim type="top">12</dim>
+                                        <dim type="bottom">25</dim>
+                                        <dim type="right">12</dim>
+                                        <dim type="left">8</dim>
+                                    </dimensions>
+                                    <note>The number of characters per line per column is 12–25
+                                        (<locus target="#3r"/>).</note>
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling, the
+                                        bottom line is written above the ruling.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A/0-0/0-0/C</ab>
+                                </layout>
+                                
+                                <layout columns="2" writtenLines="22">
+                                    <locus from="144r" to="161vb"/>
+                                    <note corresp="#textarea #margin">Data on text area and margin
+                                        dimensions taken from <locus target="#150r"/>.</note>
+                                    <dimensions unit="mm" xml:id="textarea">
+                                        <height>75</height>
+                                        <width>57</width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin">
+                                        <dim type="top">12</dim>
+                                        <dim type="bottom">24</dim>
+                                        <dim type="right">14</dim>
+                                        <dim type="left">7</dim>
+                                        <dim type="intercolumn">5</dim>
+                                    </dimensions>
+                                      <note>The number of characters per line per column is 9–10.</note>
+                                    <ab type="pricking">Pricking and ruling are visible.</ab>
+                                    <ab type="ruling"> The upper line is written above the ruling, the
+                                        bottom line is written above the ruling.</ab>
+                                    <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C</ab>
+                                </layout>
+                                
+                            </layoutDesc>
                         </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Broad and rounded letter shapes. The vowel marker of the 
```

Comment:
`the fifth order and of the sixth-order <foreign xml:lang="gez">ል</foreign>` – this sounds a bit strange to me. Maybe `the fifth-order <foreign xml:lang="gez">ሌ</foreign> and of the sixth-order <foreign xml:lang="gez">ል</foreign>` or `the fifth and sixth orders of <foreign xml:lang="gez">ለ</foreign>` ?

Reply:
Ah, it sounded strange because you were saying something different from what I understood, hehe! Now it is clearer